src/dev/platform/f3xx/RTCf3xx.cpp

```diff
@@ -1,24 +1,27 @@
-#include <EVT/dev/platform/f3xx/f302x8/RTC302x8.hpp>
+#include <EVT/dev/platform/f3xx/RTCf3xx.hpp>
 
 #include <HALf3/stm32f3xx_hal_rtc.h>
 
-namespace EVT::core::DEV {
-
-RTCf302x8::RTCf302x8() {
-    halRTC.Instance = RTC1;
+namespace time = EVT::core::time;
 
-    // Numbers generated from STMCubeMX
-    halRTC.Init.HourFormat = RTC_HOURFORMAT_24;
-    halRTC.Init.AsynchPrediv = 127;
-    halRTC.Init.SynchPrediv = 255;
-    halRTC.Init.OutPut = RTC_OUTPUT_DISABLE;
-    halRTC.Init.OutPutType = RTC_OUTPUT_TYPE_PUSHPULL;
-    halRTC.Init.OutPutPolarity = RTC_OUTPUT_POLARITY_HIGH;
+namespace EVT::core::DEV {
 
+RTCf3xx::RTCf3xx() : halRTC{
+    RTC1,
+    {
+        // Numbers generated from STMCubeMX
+        RTC_HOURFORMAT_24,
+        127,
+        255,
+        RTC_OUTPUT_DISABLE,
+        RTC_OUTPUT_TYPE_PUSHPULL,
+        RTC_OUTPUT_POLARITY_HIGH,
+    },
+} {
```

Comment:
Like what Matt mentioned above, this formatting looks a little scuffed

Reply:
I'm not really sure how this works, but CLion really wants me to do this, and it seems to make stuff a little more explicit, so I'm okay with it